lib/galaxy/managers/folders.py

```diff
@@ -171,7 +188,7 @@ def update(self, trans, folder, name=None, description=None):
         """
         changed = False
         if not trans.user_is_admin:
-            if not self.check_manageable(trans, folder):
+            if not self.check_modifyable(trans, folder):
```

Comment:
The raise below is unreachable, so you could drop this an just do
`self.check_modifyable(trans, folder)`

Reply:
done